Makefile

```diff
@@ -61,6 +61,11 @@ ifeq ($(TAG),)
   $(error "TAG cannot be empty")
 endif
 
+BUILD_ENV ?= latest
```

Comment:
Wouldn't `BUILD_IMAGE_VERSION` be better?

Reply:
This is passed in when the make docker command is executed. Will BUILD_IMAGE_VERSION be a bit long?
Although it's more accurate.